modules/imaging_uploader/README.md

```diff
@@ -1,9 +1,21 @@
-To enable the Imaging Uploader to handle large files, please update the apache configuration file
-with the following values.
+# Imaging Uploader
 
-File to update : `/etc/php5/apache2/php.ini`
+## Purpose
 
-Sample values: 
+The imaging uploader is intended to allow users to upload, browse, and track 
+pipeline insertion progress.
+
+
+## Requirements
+
+A successful scan upload should obey the following requirements: 
```

Comment:
Doesn't the imaging_uploader just call the imaging pipeline, not do it in the module itself?

Reply:
the 2 subsections in the requirements section are actually done in imaging uploader itself. what is NOT done in the module but in the pipeline is documented in the `NOT in scope` section.

Reply:
Oh, I see. It read to me like it was a '## Requirements' section that was incomplete and just hadn't been completed and was missing the list of requirements. I didn't realize the two subsections below that were supposed to be the requirements.